.env.example

```
@@ -8,3 +8,13 @@ AUTH_SECRET=some-secret
 INTERNAL_API_BASE_URL=http://localhost:3000
 MOCK_AUTH=true
+
+# generate: openssl rand -hex 32
+ALTCHA_HMAC_KEY=change-this-to-random-string
+
+# MailDev: docker run -d -p 1025:1025 -p 8025:1080 maildev/maildev
+SMTP_FROM=forms@example.com
+SMTP_HOST=localhost
+SMTP_PASS=
+SMTP_PORT=1025
+SMTP_USER=
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)